app/transformers/api/v3/patient_transformer.rb

```diff
@@ -73,5 +73,14 @@ def to_nested_response(patient)
           end
         )
     end
+
+    private
+
+    def address_attributes(patient)
+      return Api::V3::Transformer.to_response(patient.address) if patient.address
+
+      Statsd.instance.increment("sync.missing_address")
+      {}
```

Comment:
We'll need to check if the app can deal with empty address objects. It's possible if we sync an empty address that the app crashes.

We haven't seen any errors on production since we undiscarded the addresses, I think we should probably just fix the data on all envs and introduce the address constraint. wdyt?